Parser.py

```diff
@@ -15,8 +17,9 @@ class Parser(object):
 
     def __init__(self, output, *, software=None, to_console=True,
                  to_file=False, log_file="CCParser.log", to_json=False,
-                 json_file="CCParser.json", overwrite_json=True,
-                 overwrite_vals=True, use_numpy=True):#cf. PEP-3102
+                 json_file="CCParser.json", large_fn="matrices.npz",
+                 thresh=3, overwrite_file=True, overwrite_vals=True,
```

Comment:
`thresh` is a little too general, perhaps use `split_thresh` instead. Also, `large_fn` doesn't fit the current convention for filenames. What do you think of using `large_file`, `json_split_file`, `split_file` or `npz_file` instead?